src/main/scala/mesosphere/mesos/TaskBuilder.scala

```diff
@@ -197,6 +196,7 @@ object TaskBuilder {
     // args take precedence over command, if supplied
     app.args.foreach { argv =>
       builder.setShell(false)
+      builder.setValue(argv.head)
```

Comment:
I created a Mesos JIRA ticket to fix the documentation (at least):

https://issues.apache.org/jira/browse/MESOS-2859